code/game/objects/items/tools.dm

```diff
@@ -457,6 +457,12 @@
 		flamethrower_screwdriver(I, user)
 	else if(istype(I, /obj/item/stack/rods))
 		flamethrower_rods(I, user)
+	else if(istype(I, /obj/item/reagent_containers) && (I.container_type & OPENCONTAINER_1))
+		var/amountNeeded = max_fuel - get_fuel()
+		var/obj/item/reagent_containers/container = I
+		if(amountNeeded > 0 && container.reagents.has_reagent("welding_fuel"))
+			container.reagents.trans_id_to(src, "welding_fuel", amountNeeded)
+			to_chat(user, "<span class='notice'>You transfer some fuel from \the [container] to \the [src].</span>")

```

Comment:
i always forget if `\the` is the correct way to do this please correct me if im wrong

Reply:
Nowadays we just do ```[container]``` and ```[src]```. ```\the``` does work still, but we tend on not using it.

Reply:
So it's fine to leave as is then

Reply:
Remove \the